numpy/core/src/multiarray/arraytypes.c.src

```diff
@@ -13,6 +13,7 @@
 #include "npy_pycompat.h"
 #include "numpy/npy_math.h"
 #include "numpy/halffloat.h"
+#include "lowlevel_strided_loops.h"
```

Comment:
yes, but looks like it is already included a few lines below so I'm removing this line.

 (perhaps it was added in another PR)